tools/packaging/packaging.psm1

```diff
@@ -1210,6 +1210,10 @@ function New-AfterScripts
             New-Item -Force -ItemType SymbolicLink -Target "/usr/lib/x86_64-linux-gnu/libcrypto.so.1.0.2" -Path "$Staging/libcrypto.so.1.0.0" > $null
         }
     }
+    elseif ($Environment.IsMacOS) {
+        $AfterInstallScript = [io.path]::GetTempFileName()
+        $packagingStrings.MacOSAfterInstallScript -f "$Link" | Out-File -FilePath $AfterInstallScript -Encoding ascii
```

Comment:
We should have a removal script so that if the user removes the package 😆, the entry gets removed.

Reply:
After we integrated this, we should update the brew formula with similar cleanup, based on their guidance.

Reply:
synced offline. It’s not supported on macOS so we’re deferring that to brew. I’ve added a comment